src/sys/wait.rs

```diff
@@ -1,58 +1,72 @@
+        /// process
         WUNTRACED,
+        /// Waits for children that have terminated.
+        #[cfg(any(target_os = "linux", target_os = "android"))]
         WEXITED,
+        /// Report the status of any continued child process specified by pid whose status has not
+        /// been reported since it continued from a job control stop.
+        #[cfg(any(target_os = "linux", target_os = "android"))]
         WCONTINUED,
-        WNOWAIT, // Don't reap, just poll status.
-        __WNOTHREAD, // Don't wait on children of other threads in this group
-        __WALL, // Wait on all children, regardless of type
+        /// Leave the child in a waitable state; a later wait call can be used to again retrieve
+        /// the child status information.
+        #[cfg(any(target_os = "linux", target_os = "android"))]
+        WNOWAIT,
+        /// Don't wait on children of other threads in this group
+        #[cfg(any(target_os = "linux", target_os = "android"))]
+        __WNOTHREAD,
+        /// Wait for all children, regardless of type (clone or non-clone)
+        #[cfg(any(target_os = "linux", target_os = "android"))]
+        __WALL,
+        /// Wait for "clone" children only. If omitted then wait for "non-clone" children only.
+        /// (A "clone" child is one which delivers no signal, or a signal other than `SIGCHLD` to
+        /// its parent upon termination.) This option is ignored if `__WALL` is also specified.
+        #[cfg(any(target_os = "linux", target_os = "android"))]
         __WCLONE,
+
     }
 );
 
-#[cfg(any(target_os = "linux",
-          target_os = "android"))]
+#[cfg(any(target_os = "linux",target_os = "android"))]
 const WSTOPPED: WaitPidFlag = WUNTRACED;
 
 #[derive(Eq, PartialEq, Clone, Copy, Debug)]
+/// Contains the status returned by the `wait` and `waitpid` functions.
 pub enum WaitStatus {
+    /// Signifies that the process has exited, providing the PID and associated exit status.
     Exited(Pid, i8),
+    /// Signifies that the process was killed by a signal, providing the PID, the associated
+    /// signal, and a boolean value that is set to `true` when a core dump was produced.
     Signaled(Pid, Signal, bool),
+    /// Signifies that the process was stopped by a signal, providing the PID and associated
+    /// signal.
     Stopped(Pid, Signal),
+    /// Signifies that the process was stopped due to a ptrace event, providing the PID, the
+    /// associated signal, and an integer that represents the status of the event.
     #[cfg(any(target_os = "linux", target_os = "android"))]
     PtraceEvent(Pid, Signal, c_int),
+    /// Signifies that the process received a `SIGCONT` signal, and thus continued.
     Continued(Pid),
+    /// If `WNOHANG` was set, this value is returned when no children have changed state.
     StillAlive
 }
 
-#[cfg(any(target_os = "linux",
-          target_os = "android"))]
+#[cfg(any(target_os = "linux", target_os = "android"))]
```

Comment:
Please alphabetize the list of oses here and everywhere else `cfg` is used within the scope of this PR.